bitnami/ghost/templates/NOTES.txt

```diff
@@ -142,8 +142,3 @@ host. To configure Ghost to use and external database host:
   {{- $requiredGhostPasswordError := include "common.validations.values.single.empty" $requiredGhostPassword -}}
   {{- $passwordValidationErrors =  append $passwordValidationErrors $requiredGhostPasswordError -}}
 {{- end -}}
-{{- if .Values.mariadb.enabled }}
-    {{- $mariadbPasswordValidationErrors := include "common.validations.values.mariadb.passwords" (dict "secret" $databaseSecretName "subchart" true "context" $) -}}
-    {{- $passwordValidationErrors = append $passwordValidationErrors $mariadbPasswordValidationErrors -}}
-{{- end }}
-{{- include "common.errors.upgrade.passwords.empty" (dict "validationErrors" $passwordValidationErrors "context" $) -}}
```

Comment:
You are removing this check here rather than using the corresponding MySQL validation. This can be done as long as the MySQL subchart does no longer need the passwords to be explicitly provided when upgrading. 

Is this the case? Have you verified this works fine?